native_client/deepspeech.h

```diff
@@ -244,6 +258,12 @@ void DS_FreeStream(StreamingState* aSctx);
 DEEPSPEECH_EXPORT
 void DS_FreeMetadata(Metadata* m);
 
+/**
+ * @brief Free memory allocated for result information.
+ */
+DEEPSPEECH_EXPORT
+void DS_FreeResult(Result* r);
```

Comment:
Either we expose that with `DS_FreeMetadata` or we keep just `DS_FreeResult` and we stop exposing `DS_FreeMetadata`. I'd prefer just having `DS_FreeResult` exposed.